tests.py

```diff
@@ -22,6 +22,10 @@
 
 CLA = 0xED
 
+LEDGER_MAJOR_VERSION = 1
+LEDGER_MINOR_VERSION = 0
+LEDGER_PATCH_VERSION = 17
```

Comment:
currently, the latest patch version in the `develop` branch is `19`, not `17`

Reply:
Thanks, my branch was not up to date with develop